test/cmdlineTests/storage_layout_dyn_array/output.json

```diff
@@ -0,0 +1,4 @@
+{"contracts":{"fileA":{"A":{"storageLayout":{"storage":[{"ast_id":"3","contract":"A","label":"array1","offset":"0","slot":"0","type":"t_array(t_uint256)dyn_storage"},{"ast_id":"6","contract":"A","label":"array2","offset":"0","slot":"1","type":"t_array(t_bool)dyn_storage"}],"types":{"t_array(t_bool)dyn_storage":{"base":"t_bool","encoding":"dynamic_array","label":"bool[]","numberOfBytes":"32","numberOfSlots":"1"},"t_array(t_uint256)dyn_storage":{"base":"t_uint256","encoding":"dynamic_array","label":"uint256[]","numberOfBytes":"32","numberOfSlots":"1"},"t_bool":{"encoding":"inplace","label":"bool","numberOfBytes":"1","numberOfSlots":"1"},"t_uint256":{"encoding":"inplace","label":"uint256","numberOfBytes":"32","numberOfSlots":"1"}}}}}},"errors":[{"component":"general","formattedMessage":"fileA:1:1: Warning: Source file does not specify required compiler version!
```

Comment:
Argh... we should really insert line breaks in these or pretty print in some way... do we have an issue for that?

Reply:
https://github.com/ethereum/solidity/issues/7665

Reply:
Yeah...